docs/developer/tutorials/snmp/tools.md

```diff
@@ -0,0 +1,44 @@
+# Tools
+
+## Using `tcpdump` with SNMP
+
+`tcpdump` can help visualize the exact request/response content of SNMP `GET`/`GETNEXT`/etc calls.
+
+In a shell run `tcpdump`:
+
+```
+tcpdump -vv -nni lo0 -T snmp host localhost and port 161
+```
+
+- `-nn`:  turn off host and protocol name resolution (to avoid generating DNS packets)
+- `-i INTERFACE`: listen on INTERFACE (default: lowest numbered interface)
+- `-T snmp`: type/protocol, snmp in our case
+
+
+In another separate shell run `snmpwalk` or `snmpget`:
+
+```
+snmpwalk -O n -v2c -c <COMMUNITY_STRING> localhost:1161 1.3.6.1.2.1.1.2.0
```

Comment:
Is the syntax for `snmpget` exactly the same? Or do we need another example here to show it?

Reply:
Yes, it's very similar.

```suggestion
snmpwalk -O n -v2c -c <COMMUNITY_STRING> localhost:1161 1.3.6.1.2.1.1.2.0
```

Reply:
The command for `snmpget` is like `snmpget -O n -v2c -c <COMMUNITY_STRING> localhost:1161 1.3.6.1.2.1.1.2.0`